doc/adr/0008-concept-for-handling-ldml.de-in-backend.md

```diff
@@ -0,0 +1,53 @@
+# 6. Concept for handling LDML.de in the backend
+
+Date: 2024-04-12
+
+## Status
+
+Proposed
+
+## Context
+
+We have identified two main issues with our current approach in the backend source code:
+- The current domain model is quite artificial, depicting just one specific use case (a.k.a amending laws).
+- The database model contains data that is duplicated, meaning present within the XML (xml columns) and in a separate column.
+- LDML.de is not the source of truth
+
+This ADR outlines a general concept for how to handle the XMLs in LDML.de to tackle the mentioned issues above.
+
+## Decision
+
+This concept consists of:
+1. The database model will be re-designed. We won't have anymore `amending_law`, `article` and `target_law` but rather one table for process data
+and another table for the work data. The latter will contain two column (`eli` and `xml`).
```

Comment:
During our discussion, I got used to calling it "work" data, but initially I did not understand what "work" should refer to.
Maybe "laws" or "documents" or something similar would be more intuitive?

Reply:
We call that data work data to make the differentiation with the process data. Work data was always pictured as the data containing the xml files. But sure the table could be called as we best find fit, maybe "norms"?